tests/src/Common/CoreFX/CoreFX.depproj

```diff
@@ -84,9 +52,22 @@
     <PackageReference Include="System.Data.Odbc">
       <Version>$(MicrosoftPrivateCoreFxNETCoreAppVersion)</Version>
     </PackageReference>
-    <PackageReference Include="System.Data.SqlClient">
+    <PackageReference Include="runtime.native.System.Data.SqlClient.sni">
+      <Version>$(MicrosoftPrivateCoreFxNETCoreAppVersion)</Version>
+    </PackageReference>
+    <PackageReference Include="System.Security.Principal.Windows">
       <Version>$(MicrosoftPrivateCoreFxNETCoreAppVersion)</Version>
     </PackageReference>
+    <PackageReference Include="System.Text.Encoding.CodePages">
+      <Version>$(MicrosoftPrivateCoreFxNETCoreAppVersion)</Version>
+    </PackageReference>
+    <PackageReference Include="System.Diagnostics.DiagnosticSource">
+      <Version>$(MicrosoftPrivateCoreFxNETCoreAppVersion)</Version>
+    </PackageReference>
+    <PackageReference Include="System.Data.SqlClient">
```

Comment:
All these OOB packages need to have a better story. This is tracked by dotnet/corefx#34957, this might warrant a comment here.

Reply:
Also, shouldn't we add a subscription to update the version of these OOB packages? This one is generated in CoreFX and for your `MicrosoftPrivateCoreFxNETCoreAppVersion` there's `System.Data.SqlClient/4.7.0-preview6.19229.1` matching. Otherwise we might get mismatch between tests and BCL bits.

Reply:
Yes, you want a subscriptions for these until we implemented something on our side.

Reply:
@ViktorHofer do you have a quick list of the OOB packages we might use? Or can you rule out things in this list? https://gist.github.com/hoyosjs/9db582bee3cac5c4dfb1a68878eed3ef

Reply:
My guess is the platforms and the platforms and performance counters are not.

Reply:
@hoyosjs What is a subscription?

Reply:
Let me come up with a better solution than adding all the subscriptions and hardcoding the OOB packages. I can probably hack something together until next week, probably an archive with the oob assemblies, similar to the test archives that we upload.

Reply:
of course preferrably we would just have an OOB meta package...